jemalloc-sys/src/pthread_atfork.c

```diff
@@ -0,0 +1,11 @@
+#include <stdint.h>
+
+// These symbols are used by jemalloc on android but the really old android
+// we're building on doesn't have them defined, so just make sure the symbols
+// are available.
+__attribute__((weak)) int
+pthread_atfork(uint8_t *prefork __attribute__((unused)),
```

Comment:
Should use the definition from https://man7.org/linux/man-pages/man3/pthread_atfork.3.html. The original definition is just for convenient.